### ISSUE TITLE
How to export and import ExtrudeGeometry 

### ISSUE THREAD

**Issue by ghost** [mrdoob/three.js#5863]
I am trying to export and import scenes.
I use 

```
 var exporter = new THREE.SceneExporter();
 var sceneJson = JSON.stringify(exporter.parse(scene));
 localStorage.setItem('scene', sceneJson);
```

to export and

```
 var json = (localStorage.getItem('scene'));
 var sceneLoader = new THREE.SceneLoader(); 
 sceneLoader.parse(JSON.parse(json), function (e) {
         scene = e.scene;
  }, '.');
```

to import.

The scene I export has SphereGeometry and ExtrudeGeometry but when I import the scene I can only see the spheres.
I checked the exported json file using Blob and found that there are no extrudeSettings in the file, I guess maybe that is the point.

How can I import or export in this case?

Thanks!

**Comment by mrdoob** [comment]
Neither `SceneExporter` nor `ObjectExporter` support `ExtrudeGeometry` yet...

**Comment by saintni** [comment]
Please tell me when it is supported.
Thanks for the quick reply.

Best regards
Shitian Ni

**Comment by ekatzenstein** [comment]
Yes, this would be very helpful.

**Comment by ekatzenstein** [comment]
FYI, the below worked for me to create new geometry from the extruded geometry.

```
geo=new THREE.Geometry();
geo.vertices=extrudedGeometry.vertices;
geo.faces=extrudedGeometry.faces;
```

**Comment by Mugen87** [comment]
This is now fixed via #13811 :tada:
